bluebottle/clients/management/commands/bulk_import.py

```diff
@@ -54,18 +53,35 @@ def add_arguments(self, parser):
                             help="JSON import file.")
         parser.add_argument('--tenant', '-t', action='store', dest='tenant',
                             help="The tenant to run the recurring donations for.")
+        parser.add_argument('--models', '-m', action='store', dest='models',
```

Comment:
I am not sure of the lifetime of this bulk importer. If we envision keeping it around even after the vjb migration, then it would be nice to validate the inputs via the command line. see here: [e.g.](https://github.com/onepercentclub/bluebottle/blob/master/bluebottle/analytics/management/commands/export_engagement_metrics.py#L59)